packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts

```diff
@@ -303,11 +303,13 @@ export class OdspDocumentServiceFactoryCore
 			this.getWebsocketToken === undefined
 				? undefined
 				: async (options: TokenFetchOptions): Promise<string | null> =>
+						// websocket expects a plain token
```

Comment:
Could you explain what is a plain token?

Reply:
Done in jsdoc on top of the function